rules/macos/execution_installer_package_spawned_network_event.toml

```diff
@@ -64,9 +64,9 @@ tags = ["Domain: Endpoint", "OS: macOS", "Use Case: Threat Detection", "Tactic:
 type = "eql"
 
 query = '''
-sequence by host.id, user.id with maxspan=30s
-[process where host.os.type == "macos" and event.type == "start" and event.action == "exec" and process.parent.name : ("installer", "package_script_service") and process.name : ("bash", "sh", "zsh", "python", "osascript", "tclsh*")]
-[network where host.os.type == "macos" and event.type == "start" and process.name : ("curl", "osascript", "wget", "python")]
+sequence by host.id with maxspan=15s
+[process where host.os.type == "macos" and event.type == "start" and event.action == "exec" and process.parent.name : ("installer", "package_script_service") and process.name : ("bash", "sh", "zsh", "python", "osascript", "tclsh*")] by process.entity_id
```

Comment:
Any chance `process.name` could be any of the following be added since we already list `python`.

* `ruby`
* `go`
* `php`
* `perl`
* `java`

For network connections, does it make sense to include SCP and NC being used?

Reply:
php, perl, go, scp and netcat aren't really valid here due to the way threat actors approach second stage payload downloads and what is installed by default on macOS. Ruby and Java could be added.